service/configuration.ini

```diff
@@ -0,0 +1,4 @@
+[DEFAULT]
+
+[consume_messages_on_fail]
```

Comment:
wrong structure for an ini, needs to be:

```ini
[DEFAULT]
consume_messages_on_fail=True
```

We're also not using it at the moment.

Reply:
fixed